frontend/src/app/shared/components/editor/components/ckeditor-augmented-textarea/ckeditor-augmented-textarea.component.ts

```diff
@@ -288,23 +301,30 @@ export class CkeditorAugmentedTextareaComponent extends UntilDestroyedMixin impl
   }
 
   private setLabel() {
-    const label = document.querySelector<HTMLLabelElement>(`label[for=${this.textAreaId}]`)!;
-
     const ckContent = this.element.querySelector<HTMLElement>('.ck-content')!;
+    const label = document.querySelector<HTMLLabelElement>(`label[for=${this.textAreaId}]`);
 
-    let labelId;
-    if (label.hasAttribute('id')) {
-      labelId = label.getAttribute('id')!;
-    } else {
-      labelId = uniqueId('label-');
-      label.setAttribute('id', labelId);
-    }
+    if (label) {
+      let labelId;
+      if (label.hasAttribute('id')) {
+        labelId = label.getAttribute('id')!;
+      } else {
+        labelId = uniqueId('label-');
+        label.setAttribute('id', labelId);
+      }
 
-    ckContent.removeAttribute('aria-label');
-    ckContent.setAttribute('aria-labelledby', labelId);
+      ckContent.removeAttribute('aria-label');
+      ckContent.setAttribute('aria-labelledby', labelId);
 
-    fromEvent(label, 'click')
-      .pipe(this.untilDestroyed())
-      .subscribe(() => ckContent.focus());
+      if (!this.labelClickSubscription) {
+        this.labelClickSubscription = fromEvent(label, 'click')
+          .pipe(this.untilDestroyed())
+          .subscribe(() => ckContent.focus());
+      }
+    } else if (this.textAreaAriaLabel) {
+      ckContent.setAttribute('aria-label', this.textAreaAriaLabel);
+    } else {
+      throw new Error(`Please provide a label for the textarea with id ${this.textAreaId} or set the textAreaAriaLabel property.`);
```

Comment:
I think we should reserve throwing Errors for critical failures, and instead go for `console.error` or `console.warn` (thus, not halting execution):

```suggestion
      console.error(`Please provide a label for the textarea with id ${this.textAreaId} or set the textAreaAriaLabel property.`);
```

(although I'd love a second opinion on this comment - @HDinger or @oliverguenther?)